src/frontend/src/features/blocknote/blocknote-view-field/_index.scss

```
@@ -47,88 +47,94 @@
-        border-radius: 0;
-        border-top-left-radius: var(--c--components--forms-input--border-radius);
-        border-top-right-radius: var(--c--components--forms-input--border-radius);
+    // Nested under `.composer-field` so the rule does NOT apply to the
+    // detached portal `<div>` BlockNote >=0.48 appends to `document.body`
+    // (it carries the same `composer-field-input` class but has no parent
+    // `.composer-field`). Without this scope, the portal renders as an
+    // empty ~320px bordered block at the bottom of the page.
+    .composer-field-input {
+        --min-height: 20rem;
+        --toolbar-height: 43px;
+        border-radius: var(--c--components--forms-input--border-radius);
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)